### ISSUE TITLE
github reports "Error: direct image runtime not supported"

### ISSUE THREAD

**Issue by joequant** [containers/libpod#6070]
/kind bug

**Description**

podman version 0fd8242 (2020 May 2) reports "Error: direct image runtime not supported"

Works for version 8857ba2 (2020 April 15)

**Steps to reproduce the issue:**

1.  Run podman

**Describe the results you received:**

All runs give erro

**Additional information you deem important (e.g. issue happens only occasionally):**

**Output of `podman version`:**

```
(paste your output here)
```

**Output of `podman info --debug`:**

```
[joe@big-apple RPMS]$ podman info --debug                                       │··············································
Error: direct image runtime not supported                                       │··············································
[joe@big-apple RPMS]$  
```

**Package info (e.g. output of `rpm -q podman` or `apt list podman`):**

```
(paste your output here)
```

**Additional environment details (AWS, VirtualBox, physical, etc.):**

**Comment by joequant** [comment]
This seems to be triggered by 

https://github.com/containers/libpod/commit/554c663b5ad2bd0c0fa4c58559ec2a9b19ed399f

**Comment by rhatdan** [comment]
This looks like you are running podman-remote not podman?  Is this the rpm installed podman?
Did you use something like CRC to install podman?

**Comment by joequant** [comment]
I'm a packager for mageia and this is what happens when generate the rpm.  When I generate the rpm with sources before mid-April, the works.  When I run the spec file against May sources, it doesn't.

**Comment by joequant** [comment]
Here is the spec file.  Could something have changed with the build 

[podman.spec.txt](https://github.com/containers/libpod/files/4590987/podman.spec.txt)

**Comment by joequant** [comment]
The specific commit that causes problems is 241326a9a8c20ad7f2bcf651416b836e7778e090

Maybe some interaction with build script?

**Comment by joequant** [comment]
Or is there some migration between v1 and v2?

**Comment by joequant** [comment]
Think I found it.  I'm running go build directly and so the bulld args changed between V1 and V2

**Comment by joequant** [comment]
Yeah.  That's the problem.  Closing

**Comment by rhatdan** [comment]
Greate.  @joequant There is huge changes going on in Master branch.  So you need to realize this.  If you want to go with the released branch `v1.9`, that should be more stable.

We are just finishing up getting podman V2.0 to pass `rootful` tests.  Next we will get it to pass `rootless` tests, followed by more distributions, and then on to making sure podman-remote is passing all tests.

**Comment by mheon** [comment]
In short, master is really not stable right now, so use at your own risk. We'll start cutting release candidates for v2.0 once we feel master is stable again, but that's probably a few weeks off.
